### PR TITLE
add sav rate by product

### PR DESCRIPTION
**Pull request by fixo94**
Add  % of sav rate by product

### CODE REVIEW

File: statsbestproducts.php [hunk 1]
```diff
@@ -107,6 +107,12 @@ public function __construct()
                 'header' => $this->trans('Active', array(), 'Admin.Global'),
                 'dataIndex' => 'active',
                 'align' => 'center'
+            ),
+	    array(
+                'id' => 'savrate',
+                'header' => $this->trans('% retour', array(), 'Modules.Statsbestproducts.Admin'),
```
Comment (jf-viguier) [inline statsbestproducts.php]: in english please

Comment (fixo94) [reply statsbestproducts.php]: Sure, done !

File: statsbestproducts.php [hunk 2]
```diff
@@ -191,6 +197,16 @@ public function getData()
         foreach ($values as &$value) {
             $value['avgPriceSold'] = Tools::displayPrice($value['avgPriceSold'], $currency);
             $value['totalPriceSold'] = Tools::displayPrice($value['totalPriceSold'], $currency);
+	    $q = "SELECT count(oret.id_order_return) AS nr FROM "._DB_PREFIX_."order_return AS oret
+LEFT JOIN "._DB_PREFIX_."order_return_detail AS ord ON (oret.id_order_return=ord.id_order_return)
+LEFT JOIN "._DB_PREFIX_."order_detail AS od ON (ord.id_order_detail=od.id_order_detail)  
+LEFT JOIN "._DB_PREFIX_."product_attribute AS pa ON (od.product_attribute_id=pa.id_product_attribute)
+LEFT JOIN "._DB_PREFIX_."order_return_state_lang AS status ON (oret.state=status.id_order_return_state) 
+WHERE status.id_lang=1
```
Comment (jf-viguier) [inline statsbestproducts.php]: (int)$this->getLang()

File: statsbestproducts.php [hunk 3]
```diff
@@ -191,6 +197,16 @@ public function getData()
         foreach ($values as &$value) {
             $value['avgPriceSold'] = Tools::displayPrice($value['avgPriceSold'], $currency);
             $value['totalPriceSold'] = Tools::displayPrice($value['totalPriceSold'], $currency);
+	    $q = "SELECT count(oret.id_order_return) AS nr FROM "._DB_PREFIX_."order_return AS oret
+LEFT JOIN "._DB_PREFIX_."order_return_detail AS ord ON (oret.id_order_return=ord.id_order_return)
```
Comment (jf-viguier) [inline statsbestproducts.php:201]: please indent

File: statsbestproducts.php [hunk 4]
```diff
@@ -191,6 +197,16 @@ public function getData()
...
+LEFT JOIN "._DB_PREFIX_."order_detail AS od ON (ord.id_order_detail=od.id_order_detail)  
+LEFT JOIN "._DB_PREFIX_."product_attribute AS pa ON (od.product_attribute_id=pa.id_product_attribute)
+LEFT JOIN "._DB_PREFIX_."order_return_state_lang AS status ON (oret.state=status.id_order_return_state) 
+WHERE status.id_lang=1 
+AND date_add BETWEEN $date_between
+AND od.product_id='" . $value['id_product'] . "';";
```
Comment (jf-viguier) [inline statsbestproducts.php]:
```suggestion
AND od.product_id='" . (int)$value['id_product'] . "';";
```
Thanks to Arnaud from SC

File: statsbestproducts.php (hunk 2 above)
Comment (fixo94) [reply statsbestproducts.php]: Done

File: statsbestproducts.php (hunk 3 above)
Comment (fixo94) [reply statsbestproducts.php:201]: mmm indented with my tools :(

File: statsbestproducts.php (hunk 4 above)
Comment (fixo94) [reply statsbestproducts.php]: Done, thanks Arnaud

**Pull request closed without merging**